src/Autocomplete.elm

```diff
@@ -171,7 +171,7 @@ updateModel msg model =
         ( { model | selectedItemIndex = boundedNewIndex }, { defaultStatus | selectionChanged = True } )
 
     ShowMenu bool ->
-      ( { model | showMenu = bool }, defaultStatus )
+      ( { model | showMenu = bool }, {defaultStatus | completed = not bool})
```

Comment:
I don't think the `completed` status should change, since toggling the menu is accessible via `showMenu`. `completed` should only be `True` when a completion code is toggled, like the default code: the `tab` key.

So in other words, this Msg type `ShowMenu` will not affect whether the autocomplete is completed. Your Elm app can always call a helper function `getCurrentValue` to see if the current value matches something in the provided item list.